backend/src/test/java/moaon/backend/api/project/ProjectApiTest.java

```diff
@@ -141,4 +144,35 @@ void getAllProjects() {
                 () -> assertThat(actualResponses[2].id()).isEqualTo(projectViewRankThird.getId())
         );
     }
+
+    @DisplayName("GET /projects/{id}/articles : 특정 프로젝트 ID에 있는 모든 아티클 조회 API")
+    @Test
+    void getArticlesByProjectId() {
+        // given
+        Project targetProject = repositoryHelper.save(new ProjectFixtureBuilder().build());
+        Project otherProject = repositoryHelper.save(new ProjectFixtureBuilder().build());
+        Article targetProjectArticle1 = repositoryHelper.save(
+                new ArticleFixtureBuilder().project(targetProject).build());
+        Article targetProjectArticle2 = repositoryHelper.save(
+                new ArticleFixtureBuilder().project(targetProject).build());
+        Article targetProjectArticle3 = repositoryHelper.save(
+                new ArticleFixtureBuilder().project(targetProject).build());
+        repositoryHelper.save(new ArticleFixtureBuilder().project(otherProject).build());
+
+        // when
+        ArticleDetailResponse[] actualArticles = RestAssured.given().log().all()
+                .queryParams("id", targetProject.getId())
+                .when().get("/projects/" + targetProject.getId() + "/articles")
+                .then().log().all()
+                .statusCode(200)
+                .extract().as(ArticleDetailResponse[].class);
+
+        // then
+        assertAll(
+                () -> assertThat(actualArticles).hasSize(3),
+                () -> assertThat(actualArticles[0].id()).isEqualTo(targetProjectArticle1.getId()),
+                () -> assertThat(actualArticles[1].id()).isEqualTo(targetProjectArticle2.getId()),
+                () -> assertThat(actualArticles[2].id()).isEqualTo(targetProjectArticle3.getId())
+        );
```

Comment:
이 테스트에서 순서 까지 검증이 되어야하는게 좀 아쉬워서 찾아보니까 아래와 같이 적용할 수도 있다고 합니다.

```suggestion
        assertThat(actualArticles)
                .extracting(ArticleDetailResponse::id)
                .containsExactlyInAnyOrder(
                        targetProjectArticle1.getId(),
                        targetProjectArticle2.getId(),
                        targetProjectArticle3.getId()
                );
```

Reply:
좋은 것 같아요 !